LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Amol Grover
```

Comment:
Hi @sep2001aryan. Thanks again for your PR. One little doubt for me here is the name **Amol Grover**

Reply:
Any reason for this name?

Reply:
I forgot to change the name on license. I am getting started 😅 sorry for that

Reply:
I can change it would accept the PR ?

Reply:
No issues, @sep2001aryan. Glad you made a PR in this repo. If you need any help in contributing, please reach me out.